3_config.yml

```diff
@@ -4,19 +4,34 @@ include:
 
 packages:
   - dplyr
+  - RJSONIO
+  - zip
 
 sources:
   - src/fetch_filter_functions.R
+  - src/file_functions.R
 
 targets:
   3_config:
     depends:
-      - out_data/reservoir_nml_list.rds
-  
-  ##### Transfer and filtering of files of file from lake-temperature-model-prep #####
-  out_data/reservoir_nml_list.rds:
+      - out_data/reservoir_nml_list.json
```

Comment:
Only edit I see is that this file dependency should be named `out_data/reservoir_nml_values.json` ('list' is currently in the filename), and that the dependency list should also include `out_data/reservoir_nml_files.zip`